canonical/ubuntu-com-security-api
# Update cve status options and apply updates across test fixtures

## Done

- Updated cve statuses to replace "active" with "in-progress"

## QA

- Check out this feature branch
- Run the site using the command `dotrun`
  - If you get errors in this step, stop and restart the docker container then try `dotrun` again  
- View the site locally in your web browser at: http://0.0.0.0:8030/security/cves.json
- See that the endpoint returns without error and that the default CVE status is "in-progress" instead of "active"


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-15029
